build.gradle

```diff
@@ -49,6 +51,11 @@ subprojects {
         jcenter()
     }
 
+    def isSpotbugsEligible = ['newrelic-agent', 'newrelic-weaver', 'agent-bridge', 'agent-bridge-datastore', 'agent-model', 'agent-interfaces', 'newrelic-api'].contains(project.name)
+    if (isSpotbugsEligible) {
+        apply plugin: 'com.github.spotbugs'
+    }
+
```

Comment:
Could the apply plugin be done in `spotbugs-config.gradle`?
This way only the modules that include that file will have it and we skip this check.

Reply:
done